picotest_helpers/src/migration.rs

```diff
@@ -398,6 +398,30 @@ mod test {
         assert_eq!(parsed[5].text(), "DROP TABLE a;");
     }
 
+    #[rstest]
+    fn migration_file_parse_multiline_2() {
+        let text = r#"
+        -- pico.UP
+        CREATE TABLE auth_sessions (
+            token TEXT NOT NULL,
+            PRIMARY KEY (token)
+        )
+        USING memtx DISTRIBUTED BY (token)
+        IN TIER @_plugin_config.router_tier
+        OPTION (TIMEOUT = 3.0);
+
+        -- pico.DOWN
+        DROP TABLE auth_sessions;
+        "#;
+        let parsed = parse_migration_text(text).unwrap();
+        assert_eq!(parsed.len(), 4);
+        assert!(parsed[0].is_pico_up());
+        assert!(parsed[2].is_pico_down());
+        assert!(!parsed[1]
+            .text()
+            .contains("@_plugin_config.router_tierOPTION"));
```

Comment:
Вот думаю, мб надежнее проверять на то как должно быть вместо одного неверного кейса?
Типа
```suggestion
        assert!(parsed[1]
            .text()
            .contains("IN TIER @_plugin_config.router_tier OPTION (TIMEOUT = 3.0)"));
```